common/common.cpp

```diff
@@ -620,14 +621,17 @@ bool gpt_params_parse_ex(int argc, char ** argv, gpt_params & params) {
                 invalid_param = true;
                 break;
             }
-#ifndef GGML_USE_CUBLAS
-            fprintf(stderr, "warning: llama.cpp was compiled without cuBLAS. Setting the split mode has no effect.\n");
-#endif // GGML_USE_CUBLAS
+#ifndef GGML_USE_CUBLAS || defined(GGML_USE_SYCL)
```

Comment:
Invalid pre-processor expression. It is also unclear what you are trying to query